Index: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/VuforiaMethodsTesting.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/VuforiaMethodsTesting.java	(date 1615238705000)
+++ TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/VuforiaMethodsTesting.java	(date 1615238705000)
@@ -0,0 +1,27 @@
+package org.firstinspires.ftc.teamcode.teleops;
+
+import com.qualcomm.robotcore.eventloop.opmode.Disabled;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import org.firstinspires.ftc.teamcode.RobotPos;
+import org.firstinspires.ftc.teamcode.hardware.vision.VuforiaMethods;
+
+@TeleOp(name = "VisionPos Test", group = "testing")
+//@Disabled
+public class VuforiaMethodsTesting extends OpMode {
+    VuforiaMethods isabled = new VuforiaMethods(hardwareMap);
+    RobotPos pos;
+
+    public void init(){
+        telemetry.addLine("Reafy");
+        telemetry.update();
+        pos = new RobotPos(0,0,0);
+    }
+
+    public void loop(){
+        telemetry.addData("Target visible", isabled.targetVisible());
+        if(isabled.getPosition(pos)!=null)telemetry.addLine(""+isabled.getPosition(pos));
+        telemetry.update();
+    }
+}
